### ISSUE TITLE
:ruby and - are raising unexpected token errors

### ISSUE THREAD

**Issue by jocranford** [brigade/haml-lint#148]
I just upgraded from haml-lint 0.17.1 to 0.18.0 and am suddenly seeing these unexpected token errors. 

The code looks like this: 
```
-#
  Keep this in sync with public/404.html
  This page is used for application-level "not found" errors (such as an invalid or deleted model ID).
  public/404.html is used by the Rails framework for things such as routing errors.

:ruby
  @title = '404 Page Not Found'
  @bodyclass = 'four-zero-four'
  @message = 'Hey, you look a bit lost. Can we help?'
  @subtitle = 'Hmm, that\'s strange. We can\'t seem to find the page you\'re looking for either.'
```

The errors are: 
```
404.html.haml:1 [W] RuboCop: unexpected token kIN
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
404.html.haml:1 [W] RuboCop: unexpected token tMINUS
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
404.html.haml:1 [W] RuboCop: unexpected token tIDENTIFIER
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
404.html.haml:1 [W] RuboCop: unexpected token tIDENTIFIER
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
404.html.haml:7 [W] RuboCop: unexpected token tIVAR
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
```

We are using Ruby 2.3.1 and Rubocop 0.41.2

**Comment by sds** [comment]
Thanks for the report, @jocranford!

Fixed in fc874d440e42f2ebb50af62e6d33888ce2b3c742.
